spec/features/bars/user_views_all_bars_spec.rb

```diff
@@ -14,11 +14,24 @@
     test_bars = FactoryGirl.create_list(:bar, 3)
     visit '/'
 
-    expect(page).to have_content('Boston Bars')
+    expect(page).to have_content('Thirsty?')
     expect(page).to have_content(test_bars[0].name)
     expect(page).to have_content(test_bars[1].name)
     expect(page).to have_content(test_bars[2].name)
     expect(page.body.index(test_bars[2].name)).to be < page.body.index(
       test_bars[0].name)
   end
+
+  scenario 'user sees 10 bars per page, can change pages' do
+    test_bars = FactoryGirl.create_list(:bar, 11)
+    visit '/'
+
+    expect(page).to have_content(test_bars[9].name)
```

Comment:
I would break this test out:

`scenario 'user sees 10 bars per page`

`scenario 'user sees 11th bar on second page`

Also you want to expect to see bars 1-10 (not just the 10th) and not see the 11th.

And only see the 11th and not see 1-10 in the second test.